partner_multi_company/migrations/16.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 ForgeFlow S.L. (http://www.forgeflow.com)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+# pylint: disable=W8150
+from odoo.addons.partner_multi_company import hooks
```

Comment:
We can do it here correctly:

```suggestion
from ...hooks import fix_user_partner_companies
```

Reply:
Hello @pedrobaeza ,

I attempted to use the following import statement: `from ...hooks import fix_user_partner_companies`, but I encountered the following error:
`ImportError: attempted relative import with no known parent package`.

I reviewed other modules that utilize functions from hooks in their migration scripts, and they all employ absolute imports, disabling pylint. Perhaps you have encountered this error before and have found a solution?

Reply:
OK, don't worry. Let's go this way. Maybe the migration manager is not doing imports 100% Python compatible.